app/javascript/packs/store/sql_store.jsx

```diff
@@ -0,0 +1,12 @@
+import create from 'zustand'
+
+const useSqlStore = create(set => ({
+  sql: 'SELECT * FROM SOURCES',
```

Comment:
is it some default value? like for placeholder?

Reply:
It's just a prepared global state for storing the SQL - right now it does nothing. :smile: